src/order/filter/small_sets.lean

```diff
@@ -14,8 +14,8 @@ containing all powersets of members of `f`.
 `g` converges to `f.small_sets` if for all `s ∈ f`, eventually we have `g x ⊆ s`.
 
 An example usage is that if `f : ι → ℝ` is a family of nonnegative functions with integral 1, then
```

Comment:
Surely `f : ι → ℝ` doesn't have enough indices to be a family?

Reply:
```suggestion
An example usage is that if `f : ι → E → ℝ` is a family of nonnegative functions with integral 1, then
```